interfaces/HO-Portal/package.json

```diff
@@ -48,6 +48,7 @@
     "@swimlane/ngx-datatable": "^20.1.0",
     "core-js": "2.6.8",
     "file-saver": "^2.0.2",
+    "moment": "^2.29.4",
```

Comment:
What are you using 'moment' library for? This is for date/time operations right? I believe we should be consistent in the 121 platform on how we handle dates, and not use it some places and not at others.

Reply:
We are using moment library for date/time also in other interfaces, but were not yet using in HO, I think it is best approach for date formatting and takes less space and less complexity that's why I implemented it also in HO interface.

Reply:
I don't see the moment library in the other interfaces?

Reply:
Also, moment is not actively supported anymore: "Moment.js is a legacy project, now in maintenance mode. In most cases, you should choose a different library."
If we really need this we should use something like date-fns or luxon

Reply:
OK, so please revert this then Agnese.

Reply:
@agnesesaulite where are you even using it? I don't see it anywhere?